modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -16837,6 +16846,7 @@ there-are-no-x-orders=There are no {0} orders.
 there-are-no-x-related-issues=There are no {0} related issues.
 there-are-one-or-more-mandatory-vocabularies-assigned-to-the-knowledge-base-article=There are one or more mandatory vocabularies assigned to the knowledge base article.
 there-are-other-redirects-pointing-to-the-source-url-of-this-redirect=There are other redirects pointing to the source URL of this redirect. This will create a redirect chain so that users will be redirected multiple times before reaching the destination.
+there-are-selected-vocabularies-you-do-not-have-permission-to-view=There are selected vocabularies you do not have permission to view.
```

Comment:
Does this make sense?
> To edit this configuration you must have permission to view all the selected vocabularies.

Reply:
This may be clearer:
> You cannot edit this configuration because you do not have view permission on all the selected vocabularies.

Reply:
well the user actually can edit the configuration even if they dont have permission to view all vocabularies... so we cant say they cant... i think this text is sufficient. its more of an info than a warning that lets them know results might not exactly match what they have configured because there are additional vocabs in play... but i dont really know how this works. if they dont have permission to view the vocabs then i dont think they would see the additional categories in the results anyway, so maybe its not even necessary to give the user this information...

Reply:
Sounds like a candidate for at least a Learn More link. We'll need to explain how this interacts with the user's permissions, and why a certain search admin might note be seeing the same thing as the end user or a user with higher administrative privileges.

What about just giving a general info message to all users of the configuration screen?

> You can only see and configure categories if you have view permissions on the category. Learn More.
OR
> You must have view permission on a category to see and configure it in this screen. Learn More.

Reply:
sounds good, i will just switch category to vocabulary, because it it vocabularies that are being selected
>You must have view permission on a vocabulary to see and configure it in this screen. Learn More

Reply:
actually, @oliv-yu this message is "conditional" right? so it only displays when there are selected vocabs that the current user doesnt have permission to view, like the original message says

Reply:
So do I understand right?


1. Vocabulary Filtering is enabled in the Cat Facet.
1. An admin user that does not have view permission on all vocabularies in the Site comes along to configure the Cat Facet widget.
2. This message appears because by default all the available vocabularies are selected.

AND

1. The omni admin user comes along and configures the Cat Facet to display some vocabularies.
2. An admin user that does not have view permission on some of the vocabularies in the Site comes along to configure the Cat Facet widget
3. This message appears because some of the vocabularies this user cannot see are selected.

Reply:
@rbohl ya i think you got it. the vocabulary selector is "permission aware" so if a site admin with limited permissions cant see all vocabs, and an omni admin had previously selected some of those non-viewable vocabs, the site admin will not see them, which means they wont know they are already selected, therefore the message

Reply:
@BryanEngler Yep, that message shows up if the list of initial selected vocabulary IDs has any values that are not in the list of vocabulary IDs that the user can view. Then again, if the user selects all vocabs, that would mean the list is just an empty string so the message won't show up (is that ok?).

Reply:
ya i think thats fine... probably a better UX would have been to always show the tree and just have a select all button instead of an "all vocabs" vs "select vocabs" radio because it sounds like the "all vocabs" selection would let a user with less privileges undo selections (intentionally or un-intentionally) that an omni admin made? probably overthinking it but it sounds like that might have been safer

Reply:
So if the warning is conditional maybe use this?
> One or more selected vocabularies cannot be displayed because you do not have the necessary view permission. Learn More.

Reply:
ya that sounds good, will go with that. thanks @rbohl!